### DISCUSSION TITLE
Safe minimum amount of connections

### DISCUSSION THREAD

**Discussion by momala454** [Chia-Network/chia-blockchain#4712] [General]
I have a problem to "open the port", eventhough the port is working when i connect from another node to this ip + port, it doesn't seems to automatically add nodes.
Right now i have 7 full nodes connected, is it safe enough or do I need more ?

What can I do to to fix the discovery of nodes ? The full node is behind a nat and the port is redirected to it

**Comment by SirVladislav** [comment]
I got 6 full node connections at main device. I don't know if that's enough ?!
And the same situation with a port connection on second device.

If you have more than 1 device connecting to your wallet, you need to connect them correctly -> https://www.youtube.com/watch?v=_WD9ZF62kvs

Also you can check connections status at INFO logs.

**Comment by chiabug** [comment]
The number of connections required to keep the mode synced may be fairly low depending on the peers that are connected.  The issue seems to be that a low number of connections will not allow you to hear challenges and respond with a proof in a fast enough time frame that the proof can win the award for the block.  Can anyone elaborate further?

When you have a low connection count it often means that port 8444 is not open.  Your connections are limited to the peers where your side initiates the contact.  Such connections allow you to stay synced to the height of the other peer.  But because 8444 is not open, and forwarding to your machine, it means that other peers cannot initiate contact with you.  The low connection count means that you are only communicating with a small portion of the Chia network.  This means that your likelihood of winning a block challenge are reduced.
